azure-pipelines.yml

```diff
@@ -274,6 +274,14 @@ stages:
             BITS: 64
             NPY_USE_BLAS_ILP64: '1'
             OPENBLAS_SUFFIX: '64_'
+          Python38-64bit-clang:
+            PYTHON_VERSION: '3.8'
+            PYTHON_ARCH: 'x64'
+            TEST_MODE: full
+            BITS: 64
+            NPY_USE_BLAS_ILP64: '1'
+            OPENBLAS_SUFFIX: '64_'
+            CLANG_CL: 1
```

Comment:
It is surprisingly slow to build - ~3.5x slower than MSVC in CI. Is that expected?

Reply:
Locally it is also slower than clang on unix. OTOH it probably does a better job optimizing some numeric code than msvc.

Reply:
It was building twice (using build and build_ext)-- now 4:40 vs 2:59, so not that much slower.

Reply:
Ah, that's much better, thanks.